portal/src/graphql/portal/AnalyticsScreen.tsx

```diff
@@ -315,7 +315,7 @@ const AnalyticsScreenContent: React.VFC = function AnalyticsScreenContent() {
     <>
       <CommandBarContainer isLoading={loading} primaryItems={primaryItems}>
         <ScreenContent layout="list">
-          <ScreenTitle>
+          <ScreenTitle className={styles.widget}>
```

Comment:
I searched `<ScreenTitle>` and got this result. Do we need to fix all of them?

```
 $ rg '<ScreenTitle>'
portal/src/graphql/portal/GetStartedScreen.tsx
167:    <ScreenTitle>

portal/src/graphql/portal/UISettingsScreen.tsx
577:          <ScreenTitle>

portal/src/components/applications/OAuthClientSAMLForm.tsx
449:              <ScreenTitle>

portal/src/graphql/portal/DesignScreen/DesignScreen.tsx
976:          <ScreenTitle>

portal/src/graphql/portal/AnalyticsScreen.tsx
318:          <ScreenTitle>

portal/src/graphql/portal/SubscriptionScreen.tsx
638:          <ScreenTitle>
```

Reply:
Let me check them

Reply:
@louischan-oursky I think they are ok. We need `styles.widget` only if it is a direct child of `<ScreenContent>` which has a grid layout with 12 columns.